tdrs-backend/docs/api/groupless_users.md

```diff
@@ -0,0 +1,75 @@
+# Groupless users list
+
+Get a list of users who have yet to be assigned to a role
+
+**Request**:
+
+`GET /v1/users/no_roles`
+
+Parameters:
+
+
+ A valid httpOnly cookie in the request header to track the users session
+
+*Note:*
+
+- Authorization Protected 
+
```

Comment:
This list should only be accessible to the OFA Admin and Superuser roles (and maybe OFA Analyst, I will confirm). We should have that in the documentation.